.buildkite/scripts/autoformat.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
nit: for scripts I slightly prefer the longer names, more clear to folks not familiar with bash and easier to google:

```suggestion
set -o errexit
set -o nounset
set -o pipefail
```